packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -117,11 +116,10 @@ contract Ethernauts is ERC721Enumerable, Ownable, ReentrancyGuard {
             revert MintPriceError({sent: msg.value, required: mintPrice});
         }
 
-        if (availableToMint() == 0) {
+        uint tokensAvailable = availableToMint();
```

Comment:
```suggestion
        uint256 tokensAvailable = availableToMint();
```

Reply:
I'm not too in favor of the explicit uint256 rule.

Reply:
Using uint256  makes the code a bit more readable as the other formats are uint8, uint128 and so on. Plus I've read that the keccak of methods id is different.
For example if doing bytes4(keccak('transfer(address, uint)')), you'll get a different method sig ID than bytes4(keccak('transfer(address, uint256)')) and smart contracts will only understand the latter when comparing method sig IDs.

Reply:
I am also in favor of using explicit size.

Reply:
As far as I know, it's the same because it's the default initial size. But the size is explicit in all the variables of the contract. I made this suggestion to keep it consistent with the other variables.

Reply:
As far as I know, uint or uint256 is the same bc it's the default initial size. But the size is explicit throughout the contract. I made this suggestion to keep it consistent with the other variables.

Reply:
Ok, I will make all sizes explicit on another PR